Doppler.CDHelper.Test/AssertHelper.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.CDHelper
+{
+    public static class AssertHelper
+    {
+        public static bool GetValueAndContinue<T>(T input, out T output)
```

Comment:
It is from Relay: https://github.com/MakingSense/Doppler_Relay/blob/af1a29f1f5c335e0f7d4fb02e5e6ea3f9e835e3d/Relay.Application.Api.Test/Helpers/AssertHelper.cs#L7-L11